Source/ACE/Factories/LootGenerationFactory.cs

```diff
@@ -29,30 +26,14 @@ public static ActionChain GetSpawnChain(WorldObject inventoryItem, Position posi
             return LandblockManager.GetAddObjectChain(inventoryItem);
         }
 
-        public static Container CreateTestContainerObject(Player player, uint weenieId)
-        {
-            AceObject aceObject = DatabaseManager.World.GetAceObjectByWeenie(weenieId);
-            Container wo = new Container(aceObject);
-            player.Session.Network.EnqueueSend(new GameEventViewContents(player.Session, wo.Guid.Full));
-            return wo;
-        }
-
-        public static WorldObject CreateTestWorldObject(Player player, uint weenieId)
-        {
-            AceObject aceObject = DatabaseManager.World.GetAceObjectByWeenie(weenieId);
-            if (aceObject.ItemsCapacity >= 1)
-            {
-                return CreateTestContainerObject(player, weenieId);
-            }
-            return new GenericObject(aceObject);
-        }
-
         public static void CreateRandomTestWorldObjects(Player player, uint typeId, uint numItems)
         {
             var weenieList = DatabaseManager.World.GetRandomWeeniesOfType(typeId, numItems);
             for (int i = 0; i < numItems; i++)
             {
-                WorldObject wo = CreateTestWorldObject(player, weenieList[i].WeenieClassId);
+                WorldObject wo = WorldObjectFactory.CreateNewWorldObject(weenieList[i].WeenieClassId);
+                if (wo.WeenieType == WeenieType.Container)
+                    player.Session.Network.EnqueueSend(new GameEventViewContents(player.Session, wo.SnapShotOfAceObject()));
```

Comment:
any time a container is created in the world if you are in range you are shown the contents before opening?